### DISCUSSION TITLE
How to make file manager show encrypted partitions

### DISCUSSION THREAD

**Discussion by hektr** [lxqt/lxqt#2027] [Q&A] [answered]
I want file manager show and let me mount my luks encrypted partitions. What should I install for this?? Cryptsetup is already installed.

**Comment by stefonarch** [comment]
I have a LUKS encrypted USB HD, when connecting it it showed always the passwd dialog. For internal partitions it should work the same way. Are they shown in` computer:///` in pcmanfm-qt?

**Reply by hektr** [reply]
It worked this way on debian, but in arch it doesn't work. May be, some package is missing.

**Comment by tsujan** [comment, marked as answer]
@hektr, please use [Discussions](https://github.com/lxqt/lxqt/discussions) for your questions, not the bug tracker. I converted this one to a Q&A discussion.

As for your question, my answer is like @stefonarch's. Also, don't forget to install GVFS.

**Reply by hektr** [reply]
Thanks, gvfs was missing ☝️it is working now

**Comment by hektr** [comment]
One more stupid question! How to avoid the error 'The name org.freedesktop.secrets was not provided by any .service files'.  It appears after entering password for mounting encrypted partition. I googled it, and I think, it's needed to install gnome-keyring or may be gpg-agent? But what if I don't need to save password(forget password after unmounting partition)? Can I avoid this error without installing gnome-keyring/gpg-agent??

**Reply by tsujan** [reply]
Without setting up a `org.freedesktop.secrets` provider (like gnome-keyring or keepassxc in Arch), your passwords are immediately forgotten and you need to enter them every time they're asked. Apart from that, you could ignore the warning, which is issued by GVFS. But, sooner or later, another program might need a `org.freedesktop.secrets` provider too.

**Reply by hektr** [reply]
it shows another error now `No such object path '/org/freedesktop/secrets/aliases/session'`. I don't understand what is `org.freedesktop.secrets`. It doesn't look like application! I even can't find it's folder, there is not `org` folder on system's root.

**Reply by hektr** [reply]
it's was needed some little configuration to make gnome-keyring working on lxqt :) But keepassxc doesn't work, so I use gnome-keyring :)

**Reply by tsujan** [reply]
I can't tell about `keepassxc`; have always used `gnome-keyring` + `seahorse` because they're reliable.

**Reply by hektr** [reply]
> I can't tell about `keepassxc`; have always used `gnome-keyring` + `seahorse` because they're reliable.

yes, I also installed it with seahorse :)

**Reply by hektr** [reply]
but now journal shows `gkr-pam: unable to locate daemon control file`. It's very popular problem on forums, but there is no clear solution :)

**Reply by tsujan** [reply]
> but now journal shows gkr-pam: unable to locate daemon control file

I don't have them (I've set `MaxLevelStore`, `MaxLevelSyslog` and `MaxLevelKMsg` to `warning` in `/etc/systemd/journald.conf` but I don't know if that's the cause — lots of useless garbage are avoided in that way).
